references/gcp-sa-auth-shared-flow/sharedflowbundle/policies/EV.ParseAccessTokenResponse.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<ExtractVariables name="EV.ParseAccessTokenResponse">
+    <Source>gcp.access_token.response</Source>
+    <JSONPayload>
+        <Variable name="access_token" type="string">
+            <JSONPath>$.access_token</JSONPath>
+        </Variable>
+    </JSONPayload>
+    <VariablePrefix>private.gcp</VariablePrefix>
```

Comment:
trivial - but you could remove the prefix line since we only have one variable. just prepend to variable name